src/mlpack/methods/ann/activation_functions/inverse_quadratic_function.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file methods/ann/activation_functions/inverse_quadratic_function.hpp
+ * @author Himanshu Pathak
+ *
+ * Definition and implementation of Inverse Quadratic function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_INVERSE_QUAD_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_INVERSE_QUAD_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The Inverse Quadratic function, defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) = x^2 \\
+ * f'(x) = 2*x \\
+ * @f}
+ */
+class InvQuadFunction
+{
+ public:
+  /**
+   * Computes the Inverse Quadratic function.
+   *
+   * @param x Input data.
+   * @return f(x).
+   */
+  static double Fn(const double x)
+  {
+    return 1 /( 1 + x * x);
+  }
+
+  /**
+   * Computes the Spline function.
+   *
+   * @param x Input data.
+   * @param y The resulting output activation.
+   */
+  template<typename InputVecType, typename OutputVecType>
+  static void Fn(const InputVecType& x, OutputVecType& y)
+  {
+    y = 1 /(1 + arma::pow(x, 2));
+  }
+
+  /**
+   * Computes the first derivative of the Spline function.
```

Comment:
Now I'm confused, spline function, I thought this is the inverse quadratic function, maybe I missed something?

Reply:
Sorry I just forgot to change docs here.